domain/servicedefinition/utils.go

```diff
@@ -116,7 +118,10 @@ func getServiceDefinition(path string) (serviceDef *ServiceDefinition, err error
 			}
 			filters, err := getFiltersFromDirectory(path + "/" + subpath.Name())
 			if err != nil {
-				glog.Errorf("Error fetching filters at "+path, err)
+				plog.WithError(err).WithFields(log.Fields{
+					"path": path,
+					"subpath": subpath.Name(),
+				}).Error("rror fetching filters")
```

Comment:
Typo. I would suggest something like "Unable to fetch filters" rather than to log "ERROR: error"